ufmt/core.py

```diff
@@ -26,44 +41,45 @@ def ufmt_string(path: Path, content: str, config: Config) -> str:
     return content
 
 
-def ufmt_file(path: Path, dry_run: bool = False, diff: bool = False) -> bool:
```

Comment:
It's still extra work to compute the diff if it's not going to be displayed, consider keeping the arg.